gradle/libs.versions.toml

```diff
@@ -13,7 +13,9 @@ jackson_databind  = "2.18.2"
 snakeyaml         = "2.1"
 icu4j             = "77.1"
 supercsv          = "2.4.0"
-log4j             = "2.21.0"
+log4j             = "2.25.1"
+error_prone_annotations = "2.41.0"
```

Comment:
is there a reason for upgrading error_prone_annotations to 2.41.0 version when guava `33.2.1-jre` brings in `2.26.1` ?

Due to this mismatch, now plugins who uses guava has to remove `error_prone_annotations` so that there is no jar hell